argocd/overlays/moc-infra/applications/envs/emea/morty/cluster-management/local-storage.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: local-storage
+spec:
+  destination:
+    name: morty
+    namespace: argocd
```

Comment:
did you mean: `namespace: openshift-local-storage` ?

Reply:
Indeed, fixed. Thanks!